fontforge/sfd.c

```diff
@@ -4575,6 +4593,23 @@ return( lastap );
     return( ap );
 }
 
+static GuidelineSet *SFDReadGuideline(FILE *sfd, GuidelineSet **gll, GuidelineSet *lastgl)
+{
+    GuidelineSet *gl = chunkalloc(sizeof(GuidelineSet));
+    gl->name = SFDReadUTF7Str(sfd);
+    gl->identifier = SFDReadUTF7Str(sfd);
+    getreal(sfd,&gl->point.x);
+    getreal(sfd,&gl->point.y);
+    getreal(sfd,&gl->angle);
+    getint(sfd,&gl->color);
+    getint(sfd,&gl->flags);
+    if ( lastgl!=NULL )
```

Comment:
This doesn't make sense to me; from what I can see, `lastgl` is always `null`, so this is never hit?

Reply:
That was a really good catch. Fix incoming.

Reply:
The intent was to update lastgl when calling this function, but, by the time I was calling it, I thought that I was passing a double pointer and that the function would update it.

Reply:
@jtanx, inspect the new code and mark as resolved if it looks good.